clickhouse-admin/api/src/lib.rs

```diff
@@ -95,3 +68,19 @@ pub trait ClickhouseAdminApi {
         rqctx: RequestContext<Self::Context>,
     ) -> Result<HttpResponseOk<ClickhouseKeeperClusterMembership>, HttpError>;
 }
+
+#[dropshot::api_description]
```

Comment:
Could you add a little comment here explaining why the two are different? Thinking of my future self which would be wondering why.

Reply:
Good idea! Done in [9862199](https://github.com/oxidecomputer/omicron/pull/6837/commits/98621999699a16737ee73df59b5f17c880dfcb7c)